### DISCUSSION TITLE
能不能统一顶栏信息的样式

### DISCUSSION THREAD

**Discussion by YJL12345** [the1812/Bilibili-Evolved#3304] [功能建议 / Ideas]
动态的样式太大，无法快速浏览，我更喜欢稍后再看那种条形的
怎么只有动态的那么大，其他订阅、稍后、收藏、历史样式都是一样的
![Snipaste_2022-05-06_08-02-29](https://user-images.githubusercontent.com/42275612/167046041-817e48b8-0e98-49e1-86c1-20ccd0efcd99.png)
![Snipaste_2022-05-06_08-02-53](https://user-images.githubusercontent.com/42275612/167046050-acb4e50a-7824-450e-870f-2330fb1ded52.png)

**Comment by the1812** [comment]
列表一屏也是 6 个啊, 一样的

**Reply by YJL12345** [reply]
> 
看标题，明明是4个，而且左右分栏，没办法快速浏览

**Reply by the1812** [reply]
那就用原版的顶栏吧
